docs/business-apps/power-automate/get-started/sharepoint-actions-triggers.md

```diff
@@ -189,7 +189,7 @@ Grants access to an item or a folder in SharePoint to specific people.
 
 ### Join hub site
 
-Joins the requested site to the hub site. An Approval Token is required to complete the join successfully if that hub requires approval. If applicable, specify the same Approval Correlation Id as used in the "Set hub site join status to pending" action.
+Joins the requested site to the hub site. An Approval Token is required to complete the join successfully, if that hub requires approval. If applicable, specify the same Approval Correlation Id as used in the **Set hub site join status to pending** action.
 
 ### List folder
```

Comment:
Missing description. We can add the following:

_Returns files from the root SharePoint folder in a document library. 
**Note:** This action will be deprecated soon in favor of 'Get items' and 'Get files' actions. It is recommended to use 'Get items' and 'Get files' action instead of List Folder to get items or files from a list or library._